src/core/protocol-authorization.ts

```diff
@@ -430,21 +443,25 @@ export class ProtocolAuthorization {
    * Currently the only check is: if the write is not the initial write, the author must be the same as the initial write
    * @throws {Error} if fails verification
    */
-  private static async verifyActionCondition(tenant: string, incomingMessage: RecordsWrite, messageStore: MessageStore): Promise<void> {
-    const recordsWrite = incomingMessage as RecordsWrite;
-    const isInitialWrite = await recordsWrite.isInitialWrite();
-    if (!isInitialWrite) {
-      // fetch the initialWrite
-      const query = {
-        entryId: recordsWrite.message.recordId
-      };
-      const { messages: result } = await messageStore.query(tenant, [ query ]);
-
-      // check the author of the initial write matches the author of the incoming message
-      const initialWrite = result[0] as RecordsWriteMessage;
-      const authorOfInitialWrite = Message.getAuthor(initialWrite);
-      if (recordsWrite.author !== authorOfInitialWrite) {
-        throw new Error(`author of incoming message '${recordsWrite.author}' must match to author of initial write '${authorOfInitialWrite}'`);
+  private static async verifyActionCondition(tenant: string, incomingMessage: RecordsRead | RecordsWrite, messageStore: MessageStore): Promise<void> {
+    if (incomingMessage.message.descriptor.method === DwnMethodName.Read) {
+      // Currently no conditions for reads
```

Comment:
Optional nit: In preparation of other methods being supported in near future, probably just remove this `if` and perhaps say in comment after write handling:

`// Currently no conditions for other methods`

Reply:
Updated